cypress/integration/treeFromObject.spec.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+Cypress.Commands.add('haveChildren', (value) => {
+  cy.contains(value).children().children().should('have.length', 2);
+});
+
+describe('Tree from object app', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should have 1 unordered list with 2 list items', () => {
+    cy.get('#tree').children().children().should('have.length', 2);
```

Comment:
lets reuse haveChildren method?

Reply:
there we cy.get element by id, in haveTwoChildren we search for element by text